app/src/main/java/com/android/universe/ui/map/EventInfoPopup.kt

```diff
@@ -0,0 +1,88 @@
+package com.android.universe.ui.map
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.android.universe.model.event.Event
+import com.android.universe.ui.theme.Dimensions
+
+@Composable
+fun EventInfoPopup(event: Event, onDismiss: () -> Unit) {
+  Box(
+      modifier =
+          Modifier.fillMaxSize()
+              .background(MaterialTheme.colorScheme.onBackground.copy(alpha = 0.4f))
+              .clickable(onClick = onDismiss),
```

Comment:
Linked with [link](https://github.com/SwEnt-Universe/UniVERSE/pull/142/files#r2516615912)
You should update the modifier of the Box to apply a testag to be able to test the composable


```suggestion
  Box(
      modifier =
          Modifier.fillMaxSize()
              .background(MaterialTheme.colorScheme.onBackground.copy(alpha = 0.4f))
              .clickable(onClick = onDismiss).testTag(MapScreenTestTags.EVENT_INFO_POPUP),
```